src/zenml/cli/stack.py

```diff
@@ -681,3 +714,77 @@ def down_stack(force: bool = False) -> None:
             f"Suspending resources for active stack '{stack_.name}'."
         )
         stack_.suspend()
+
+
+@stack.command("export")
+@click.argument("stack_name", type=str, required=True)
+@click.argument("filename", type=str, required=True)
+def export_stack(stack_name: str, filename: str) -> None:
+    """Export a stack to YAML."""
+
+    # Get configuration of given stack
+    # TODO: code duplicate with describe()
+    repo = Repository()
+    stack_configurations = repo.stack_configurations
+    if len(stack_configurations) == 0:
+        cli_utils.warning("No stacks registered!")
+        return
+    try:
+        stack_configuration = stack_configurations[stack_name]
+    except KeyError:
+        cli_utils.error(f"Stack '{stack_name}' does not exist.")
+        return
+
+    # create a dict of all components in the specified stack
+    component_data = {}
+    for component_type, component_name in stack_configuration.items():
+        components = repo.get_stack_components(component_type)
+        for component in components:
+            if component.dict()["name"] == component_name:
+                component_dict = {
+                    key: value
+                    for key, value in component.dict().items()
+                    if key != "uuid" and value is not None
+                }
+                component_dict["flavor"] = component.FLAVOR
+                component_data[str(component_type)] = component_dict
+
+    # write zenml version and stack dict to YAML
+    yaml_data = {
+        "zenml_version": zenml.__version__,
+        "stack_name": stack_name,
+        "components": component_data,
+    }
+    with io.open(filename, "w", encoding="utf8") as outfile:
+        yaml.dump(yaml_data, outfile)
+
+
+@stack.command("import")
+@click.argument("stack_name", type=str, required=True)
+@click.argument("filename", type=str, required=True)
+def import_stack(stack_name: str, filename: str) -> None:
+    """Import a stack from YAML."""
+    with open(filename, "r") as yaml_file:
+        data = yaml.safe_load(yaml_file)
```

Comment:
For this and the YAML-writing above, we have `src/zenml/utils/yaml_utils.py` which should cover most of what you need.

Reply:
Yep, does exactly what I need, thanks!